ipaserver/install/plugins/update_managed_permissions.py

```diff
@@ -635,6 +635,10 @@ def update_entry(self, obj, entry, template,
 
         # Attributes from template
         bindruletype = template.pop('ipapermbindruletype', 'permission')
+        if bindruletype not in {"all", "anonymous", "self", "permission"}:
```

Comment:
I added this check to make the plugin fail earlier and to give a better error message.